packages/node_modules/@webex/plugin-meetings/src/meeting/util.js

```diff
@@ -452,13 +453,13 @@ MeetingUtil.startInternalStats = (meeting) => {
                 }
               });
             }
-          },
-          onData: () => {}
+          }
```

Comment:
These changes broke our media flow eventing last time we tried them, is this still going to be the case?

Reply:
ya reverted it . I thought rebase would fix it